src/swtpm/ctrlchannel.c

```diff
@@ -790,7 +790,7 @@ int ctrlchannel_process_fd(int fd,
             goto err_bad_input;
         }
 
-        mlp->flags = MAIN_LOOP_FLAG_USE_FD | MAIN_LOOP_FLAG_KEEP_CONNECTION |
```

Comment:
Can you prefix the title with 'swtpm:'.

Reply:
I meant primarily the title of the commit ...

Reply:
> I meant primarily the title of the commit ...

Yes, I was waiting until you decided what to do regarding the two changes (i.e. just one or both) so I could then amend the commit as needed. Would you like to merge this PR as-is with just the assignment fix, and follow up on the flag change separately?

Reply:
yes, let's make this two PRs.